lib/Devel/Cover/Report/Html_basic.pm

```diff
@@ -570,16 +570,16 @@ $Templates{summary} = <<'EOT';
 <script type="text/javascript">
 <!-- hide
 function filter_files(filter_by) {
-    var allelements = document.getElementsByTagName("tr");
+    var allElements = document.getElementsByTagName("tr");
```

Comment:
this isn't technically a spelling thing, but makes it easier to catch misspelled words...